scripts/unix_share_build.sh

```diff
@@ -89,29 +117,73 @@ function build_dagmc(){
                 -DBUILD_TALLY=OFF \
                 -DBUILD_BUILD_OBB=OFF \
                 -DBUILD_MAKE_WATERTIGHT=ON \
-                -DBUILD_SHARED_LIBS=ON \
-                -DBUILD_STATIC_LIBS=OFF \
+                -DBUILD_SHARED_LIBS=${BUILD_SHARED_LIBS} \
+                -DBUILD_STATIC_LIBS=${BUILD_STATIC_LIBS} \
                 -DBUILD_EXE=OFF \
                 -DCMAKE_POSITION_INDEPENDENT_CODE=ON \
                 -DCMAKE_BUILD_TYPE=Release \
                 $CMAKE_ADDITIONAL_FLAGS \
                 -DCMAKE_INSTALL_PREFIX=${PLUGIN_ABS_PATH}/DAGMC
-    
-    make -j$PROC
+
+    make
     make install
     cd ../..
     rm -rf DAGMC/DAGMC DAGMC/bld
 }
 
-function setup_Trelis_sdk() {
+function mac_setup_cubit_sdk() {
+    cd ${FOLDER_PKG}
+    if [ "${1}" = "17.1.0" ]; then
+        hdiutil convert ${CUBIT_PKG} -format UDTO -o trelis_eula.dmg.cdr
+        hdiutil attach trelis_eula.dmg.cdr -mountpoint /Volumes/Cubit
+        mv /Volumes/Cubit/*.app /Applications/
+        hdiutil detach /Volumes/Cubit
+        rm -rf trelis.dmg
+    elif [ "${1}" = "2020.2" ]; then
+        sudo installer -pkg ${CUBIT_PKG} -target /
+        rm -rf cubit.pkg
+    fi
+
+    cd ${CUBIT_PATH}
+    if [ "${1}" = "2020.2" ]; then
+        CUBIT_BASE_NAME="Coreform-Cubit-2020.2"
+    elif [ "${1}" = "17.1.0" ]; then
+        CUBIT_BASE_NAME="Trelis-17.1"
+    fi
+    sudo tar -xzf ${FOLDER_PKG}/${CUBIT_SDK_PKG}
```

Comment:
Should these be `$SUDO` instead of `sudo`? I can't recall.

Reply:
I used `sudo` for Mac instead of `$SUDO` because it should be required regardless of the platform you are using (docker, CI, local) where on ubuntu `sudo` is not required on docker as we run as root...

Reply:
Ah, nice. Didn't know that you don't get root access in an OSX docker image.